elements/title.html

```diff
@@ -39,7 +39,7 @@
   <pre class="code-example">
     <code>
       &lt;head&gt;
-        &lt;h1&gt;<strong>&lt;style&gt;</strong>This content is styled in CSS<strong>&lt;style&gt;</strong>&lt;h1&gt;
+        &lt;h1&gt;<strong>&lt;title&gt;</strong>The Most Excellent Title Ever!<strong>&lt;/title&gt;</strong>&lt;/h1&gt;
```

Comment:
`<h1>` is not needed in the title, since title is within `head` (which is not seen by the user).